README.md

```diff
@@ -65,12 +70,12 @@ On Mac/Unix bash this would need to be:
 The available `-connprops` arguments are:
 
 | Name | Values | Default | Description
-|:---- |:------ |:--------|:----------- | 
+|:---- |:------ |:--------|:----------- |
```

Comment:
Ah!  Sorry about this change & the one on line 78 as well.  I forgot to check that my editor didn't do this & just committed and pushed the changes to the readme.